operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/ControllerConfiguration.java

```diff
@@ -13,7 +13,7 @@
 public interface ControllerConfiguration<R extends CustomResource<?, ?>> {
 
   default String getName() {
-    return ControllerUtils.getDefaultResourceReconcilerName(getAssociatedControllerClassName());
+    return ControllerUtils.getDefaultReconcilerName(getAssociatedControllerClassName());
```

Comment:
`getAssociatedControllerClassName` should also be renamed to `getAssociatedReconcilerClassName`